content/en/blog/2021/revision-tags/index.md

```diff
@@ -0,0 +1,103 @@
+
+Once you've determined the new control plane is suitable for the rest of the `prod-stable` namespaces, you can change the tag to point to the new revision. This will enable you to update the namespaces labeled `prod-stable` to the new `1-10-0` revision without making any changes to the labels on the namespace.
+
+{{< image link="./tags-updated.png" caption="Updated revision tags" >}}
+
+## Stable revision tags in action
+
+For example, to create a new `prod-stable` tag for a revision `1-9-3`, you would run the following command:
+
+{{< text bash >}}
+$ istioctl x revision tag set prod-stable --revision 1-9-3
+{{< /text >}}
+
+You can then label your namespaces with the `istio.io/rev=prod-stable` label. Note, if you installed a `default` revision (i.e., no revision) of Istio, you will first have to remove the standard injection label:
+
+{{< text bash >}}
+$ kubectl label ns istioinaction istio-injection-
+$ kubectl label ns istioinaction istio.io/rev=prod-stable
+{{< /text >}}
+
+You can list the tags in your mesh with the following:
+
+{{< text bash >}}
+$ istioctl x revision tag list
+
+TAG         REVISION NAMESPACES
+prod-stable 1-9-3
+{{< /text >}}
+
+A tag is implemented with a `MutatingWebhookConfiguration`. You can verify a corresponding `MutatingWebhookConfiguration` has been created:
+
+{{< text bash >}}
+$ kubectl get MutatingWebhookConfiguration
+
+NAME                             WEBHOOKS   AGE
+istio-revision-tag-prod-stable   2          75s
+istio-sidecar-injector           1          5m32s
+{{< /text >}}
+
+Let's say you are trying to canary a new revision of the control plane based on 1.10.0. First you would install the new version using a revision:
+
+{{< text bash >}}
+$  istioctl install -y --set profile=minimal --revision 1-10-0
+{{< /text >}}
+
+You can create a new tag called `prod-canary` and point that to your `1-10-0` revision:
+
+{{< text bash >}}
+$  istioctl x revision tag set prod-canary --revision 1-10-0
+{{< /text >}}
+
+Then label your namespaces accordingly:
+
+{{< text bash >}}
+$ kubectl label ns istioinaction-canary istio.io/rev=prod-canary
+{{< /text >}}
+
+If you list out the tags in our system, you should see two stable tags pointing to different revisions:
+
+{{< text bash >}}
+$ istioctl x revision tag list
+
+TAG         REVISION NAMESPACES
+prod-stable 1-9-3
+prod-canary 1-10-0
+{{< /text >}}
+
+Any of the namespaces that you have labeled with `istio.io/rev=prod-canary` will be injected by the control plane that corresponds to the `prod-canary` stable tag name (which in this example points to the `1-10-0` revision). When you're ready, you can switch the `prod-stable` tag to the new control plane with:
+
+{{< text bash >}}
+$  istioctl x revision tag set prod-stable --revision 1-10-0 --overwrite
+{{< /text >}}
+
+When both the `prod-stable` and `prod-canary` tags both point to the same new revision, it may be safe to remove the old revision.
```

Comment:
Missing from this story: what happens when you've made that change?  Do workloads update automatically, or do you have to do a rolling restart?